src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -1717,6 +1787,86 @@ void valid() {
                 testValid("NISTHypothesis.valid: All KEs in model must be referenced by hypothesis");
             }
         }
+
+        // Each hypothesis graph must have at least one event or relation with at least one edge.
+        @Nested
+        class HypothesisRequiredOneEventOrRelationWithOneEdge {
+
+            @Test
+            void invalid() {
+                makeValidHypothesis(entity);
+                testInvalid("NISTHypothesis.invalid: Each hypothesis graph must have at least one " +
+                        "event or relation with at least one edge.");
+            }
+
+            @Test
+            void invalidRelationAndEventEdge() {
+                final Resource relation = makeRelation(model, getUri("relation-1"), system);
+                markImportance(makeClusterWithPrototype(model, getClusterUri(), relation, "Relation", system), 103);
+                markJustification(addType(relation, SeedlingOntology.GeneralAffiliation_APORA), justification);
+
+                final Resource invalidRelationEdge = markAsArgument(model, relation, SeedlingOntology.Conflict_Attack_Attacker,
```

Comment:
I see that the name of the test does describe that it's a Relation with an Event edge, but I was only able to figure that out after looking at all your other tests. It would be easier to see if there was a comment or the text in testInvalid() described that it was a Relation with an Event edge.

Reply:
Updated with a comment.